src/components/tasks/card/card.module.scss

```diff
@@ -384,6 +408,15 @@ $progressTextAnimation: #9babb8;
     width: 100%;
     display: flex;
     gap: 5px;
+
+    @media (max-width: 28rem) {
```

Comment:
Either use "rem" values everywhere for media queries or just "px" values.

Reply:
Have used "rem" in those modules as in card scss module wherein "rem" is used in all other places over this module and "px" in those modules like task & taskSearch wherein "px" is used on media query for all other places. Please let me know if we have to update this

Reply:
What do you think we should go ahead with? @bhtibrewal @shreya-mishra @harshith-venkatesh @iamitprakash

Reply:
@Maheima @iamitprakash @bhtibrewal @shreya-mishra @harshith-venkatesh @sahsisunny Please provide suggestions on this

Reply:
@Maheima Updated the updated media queries in rem to respective px. Please review